### PR TITLE
Add search for tags

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

This will enable https://github.com/sul-dlss/argo/issues/165

## Was the API documentation (openapi.yml) updated?
yes

## Does this change affect how this application integrates with other services?
yes, but will be tested when the corresponding argo pr is created.

### CODE REVIEW

File: app/controllers/administrative_tags_controller.rb [hunk 1]
```diff
@@ -13,6 +13,11 @@ def index
     render json: AdministrativeTags.for(item: @item)
   end
 
+  def search
+    results = TagLabel.where("tag like ?", "#{params[:q]}%").limit(10).pluck(:tag)
```
Comment (justinlittman) [inline app/controllers/administrative_tags_controller.rb:17]: ordering?

Comment (justinlittman) [inline app/controllers/administrative_tags_controller.rb:17]: Should limit be an optional param?

Comment (jcoyne) [reply app/controllers/administrative_tags_controller.rb:17]: Currently we only need 10. YAGNI

Comment (jcoyne) [reply app/controllers/administrative_tags_controller.rb:17]: Not part of the use case yet.

**Pull request merged**